scripts/funcotator/testing/compareTwoReferenceDictionaries.sh

```diff
@@ -20,7 +20,8 @@
 ###############################################################################
 
 #Setup variables for the script:
-SCRIPTDIR="$( cd -P "$( dirname "$0" )" && pwd )"
+UNALIASED_SCRIPT_NAME=$( readlink "${BASH_SOURCE[0]}" || echo "${BASH_SOURCE[0]}" )
```

Comment:
Can you put that these are unsupported?  Just to keep users away from asking questions.

Reply:
Also, I am not sure about the role of these scripts and why we need Jonn-Smith-only code.

Reply:
I did not want to lose them when my computer inevitably explodes.

I added comments in so that people could understand what they do and (hopefully) how to get them to work for them.

I'll add that they are unsupported in the header comments.